components/itemDetailLayout/itemdetail.module.css

```diff
@@ -0,0 +1,22 @@
+.content{
```

Comment:
File and Folder Formate is not correct. Instead of of itemDetail it should come in Product folder and the name should ProductDetail with no layout in suffix. change other files accordingly